File: cpp/Cprob/052.cpp
#include <iostream>
// #include <>

using namespace std;

int main()
{
	int num;
	cin >> num;
	int count = 1;
	int SOSU = 1;

	while(num != count){
		if (count % num == 0)SOSU++;
		num += 1;
	}
	SOSU %= 1000000000 + 7;
	cout << SOSU << endl;
	
}

File: python/Cprob/052.py
num = int(input())

count = 1
SOSU = 1

while num > 0:
	count *= num
	num -= 1

num = 1

while num != count:
	if count % num == 0:
		SOSU += 1

	num += 1



SOSU %= 10 ** 9 + 7 
print(SOSU)

# print(count)

